### DISCUSSION TITLE
Unable to use jQuery and tablesorter

### DISCUSSION THREAD

**Discussion by TheOrdinaryWow** [requarks/wiki#4540] [Help / Questions]
I was trying to use a jQuery plugin named `tablesorter` in a specified page. However after I tried adding js and css in the `Page properties`, nothing happens.

Here's how I did that:

In the js section, I filled
```

<!-- load jQuery and tablesorter scripts -->
<script type="text/javascript" src="https://cdn.bootcdn.net/ajax/libs/jquery/3.6.0/jquery.js"></script>
<script type="text/javascript" src="https://remote path to script/jquery.tablesorter.js"></script>

<!-- tablesorter widgets (optional) -->
<script type="text/javascript" src="https://remote path to script/jquery.tablesorter.widgets.js"></script>

$(function() {
  $("#myTable").tablesorter();
});

```
As instructed in the docs of [tablesorter](https://mottie.github.io/tablesorter/docs/index.html).

In the css section, I pasted strings from [theme.default.min.css](https://nzbcorzs.site/upload/javascripts/tablesorter/css/theme.default.min.css) also as instructed in the docs.

To make sure the configuration is fine, I copied the code from the docs.

Here's how I fill in html code (edited in html)
```

<table id="myTable" class="tablesorter">
  <thead>
  ...... (collapsed)
  </tbody>
</table>

```

Yet nothing happens, no matter how I manage to make it right, the plugin is not enabled. I'm not sure in which step did I do wrong, or it's just a bug of Wiki.js.

```
OS: Debian 10 using Docker Container
Version: 2.5.219
Node.js Version: 14.17.6
Database: PostgreSQL 11.13
```

It would be nice if someone can offer some help. Thanks for spend time reading this issue. :))

**Comment by mittkkoo** [comment]
I used another library without jQuery and successfully got tables sorted, see here: #6293
